src/Service/ProgrammesAggregationService.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace BBC\ProgrammesPagesService\Service;
+
+use BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\ClipRepository;
+use BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\CoreEntityRepository;
+use BBC\ProgrammesPagesService\Mapper\ProgrammesDbToDomain\CoreEntityMapper;
+use BBC\ProgrammesPagesService\Cache\CacheInterface;
+
+class ProgrammesAggregationService extends AbstractService
```

Comment:
I'm a little wary of piling everything into CoreEntitesService / ProgrammesService as then they end up getting HUUUUUUUUGE and unwieldy. I'm pondering the idea of having multiple Services that can all return the same type of thing. More obvious examples would be `ProgrammesAtoZService` and `ProgrammesCategoriesService` that contain service calls related to getting a-z and per-category filtered programmes.

I know I mentioned `ProgrammesAggregationService` first but perhaps something like `ProgrammesByAncestorService` might be a bit clearer with the intent - to contains service calls get a list of programmes by some common ancestor. e.g. "get all clips beneath programme X" or "get all episodes that are direct children of programme X"